tests/overrides/test_data_parallel.py

```diff
@@ -147,6 +147,7 @@ def training_step(self, batch, batch_idx):
     model = TestModel().to(device)
     model.trainer = MagicMock()
     model.trainer._running_stage = RunningStage.TRAINING
+    model.running_stage = RunningStage.TRAINING
```

Comment:
why was this added?
the LightningModule should have a property called `running_stage`
why is it not there?

Reply:
Maybe the latest changes should be undone and we should cherry-pick https://github.com/PyTorchLightning/pytorch-lightning/commit/0456b4598f5f7eaebf626bca45d563562a15887b instead

Reply:
I had a brief confusion, but then remembered that this only contains bugfix commits.
Will 1.2.3 be based on master again or continue off of 1.2.2?

Reply:
Continue off 1.2.2